README.md

```diff
@@ -10,20 +10,20 @@ Included in this demo app:
 - hooks for easy usage with React
 - redux store listening a client
 - HOC component listening a client and showing different content for authorized and unauthorized users.
-- getting API token and using it to get Profile (only when using Helsinki-Profiili MVP ).
+- getting API token and using it to get Profile
 
 Client dispatches events and trigger changes which then trigger re-rendering of the components using the client.
 
 ## Config
 
-Configs are in .env -files. Default endpoint for Helsinki-Profiili is Tunnistamo. For Suomi.fi authentication, it is plain Keycloak.
+Configs are in .env -files.
 
 Tunnistamo does not support silent login checks (it uses only sessionStorage) so REACT_APP_OIDC_AUTO_SIGN_IN must be 'false'. It renews access tokens so REACT_APP_OIDC_SILENT_AUTH_PATH must be changed to '/' to prevent errors for unknown redirect url.
 
 Config can also be overridden for command line:
 
 ```bash
-REACT_APP_OIDC_URL=https://foo.bar yarn start
+REACT_APP_OIDC_URL=https://foo.bar"
```

Comment:
There's an unpaired quotation mark.

Reply:
Fixed.

Reply:
Please fix it in the original commit that introduced the typo.

Reply:
Ah, fixed there now.